src/parser/rogue/outlaw/modules/spells/BetweenTheEyes.js

```diff
@@ -1,49 +0,0 @@
-import React from 'react';
-
-import SPELLS from 'common/SPELLS';
-import SpellLink from 'common/SpellLink';
-import { formatPercentage } from 'common/format';
-import Analyzer from 'parser/core/Analyzer';
-
-import DamageTracker from 'parser/shared/modules/AbilityTracker';
-
-import { t } from '@lingui/macro';
-
-import BetweenTheEyesDamageTracker from './BetweenTheEyesDamageTracker';
-
-class BetweenTheEyes extends Analyzer {
-  get thresholds() {
```

Comment:
Since you say that BTE should be prioritized over Dispatch in the other files - why is this file being removed instead of being reworked - surely it is still interesting to see BTE information on the checklist / some sort of statistic on this?

Reply:
I felt like anything I reworked here would just be the inverse of the dispatch information "You used BTE over Dispatch 95%" and would not add anything extra other than bloat.

Reply:
We should probably keep this, as it is a learning tool, and help newcomer Outlaw Rogues with this knowledge. And make sure the Dispatch module conveys the same message.